### PR TITLE
EDIT THE TITLE

### PR DESCRIPTION
**Pull request by muhammadhaekal**

### CODE REVIEW

**Comment by mhaekal** [comment]
test

**Review by mhaekal — APPROVED** [review]

**Comment by mhaekal** [comment]
cobaaaa

**Comment by mhaekal** [comment]
coba lagii

**Comment by muhammadhaekal** [comment]
cobaaa

**Comment by muhammadhaekal** [comment]
coba lagii

**Comment by muhammadhaekal** [comment]
terbaru ni

**Review by mhaekal — CHANGES_REQUESTED** [review]
request deh

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Tech Stack
 
-- ReactJs
+- ReactJs ss
```
Comment (mhaekal) [inline README.md:5]: cobaa

**Review by mhaekal — CHANGES_REQUESTED** [review]
test

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Tech Stack
```
Comment (mhaekal) [inline README.md:3]: coba aja deh banget

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
 
-- ReactJs
+- ReactJs ss
 - Redux ss
```
Comment (muhammadhaekal) [inline README.md:6]: mencoba comment

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 - TypeScript  
 - Styled-components
```
Comment (muhammadhaekal) [inline README.md:8]: abcdef

File: README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 ## Tech Stack
 
-- ReactJs
+- ReactJs ss
 - Redux ss
 - TypeScript
```
Comment (muhammadhaekal) [inline README.md:7]: adadada

File: README.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 ## Tech Stack
 
-- ReactJs
```
Comment (muhammadhaekal) [inline README.md:5]: ababab

**Review by muhammadhaekal — COMMENTED** [review]
qwer

File: README.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
```
Comment (muhammadhaekal) [inline README.md:2]: cobalgdong

**Pull request merged**